### DISCUSSION TITLE
unable to modify chlorinator

### DISCUSSION THREAD

**Discussion by dnikles** [tagyoureit/nodejs-poolController#758] [App and O/S Config Questions] [answered]
I've been trying unsuccessfully to make any modifications to my chlorinator over the API. I was able to do so last year. I did a docker pull last week and I think that's when my troubles started

5/11/2023, 9:25:58 PM] info: [9:25:58 PM] 192.168.1.150 PUT /config/chlorinator {"id":1,"poolSetpoint":50}
[5/11/2023, 9:25:58 PM] error: *Touch setChlorAsync Error: Cannot read properties of undefined (reading 'val')
[5/11/2023, 9:25:58 PM] error: Cannot read properties of undefined (reading 'val')

I'm able to read the values of my chlorinator, but don't seem to be able to PUT anything to them. Any ideas? I have the same issue when trying to change values through dash panel as well.

**Comment by rstrouse** [comment, marked as answer]
Double check that the model is selected.

**Comment by dnikles** [comment]
Thanks that pointed me in the right direction. It looks like I had a model
but it wasn’t associated with a body.
